packages/lib/services/interop/InteropService_Importer_Md.ts

```diff
@@ -108,11 +108,12 @@ export default class InteropService_Importer_Md extends InteropService_Importer_
 		let updated = md;
 		const markdownLinks = markdownUtils.extractFileUrls(md);
 		const htmlLinks = htmlUtils.extractFileUrls(md);
-		const fileLinks = unique(markdownLinks.concat(htmlLinks));
+		const pdfLinks = htmlUtils.extractPdfUrls(md);
+		const fileLinks = unique(markdownLinks.concat(htmlLinks).concat(pdfLinks));
 		for (const encodedLink of fileLinks) {
 			const link = decodeURI(encodedLink);
 
-			if (isDataUrl(link)) {
+			if (isDataUrl(link) || isMailTo(link) || isFilenameTooLong(link)) {
```

Comment:
When testing the library I found a bug that would pop when the note had a really long title, since all notes are linked in the parent notebook it would cause an error when we were parsing the HTML file.

Reply:
That fix should be moved to a separate PR